docs/expressions.md

```diff
@@ -176,6 +178,68 @@ lstat returns various summary stats per bucket for the specified `field`. The fi
   * As of January 15, 2015 - logstash functionality is new so these functions may change a fair amount based on experience using them in alerts.
   * Alerts using this information likely want to set ignoreUnknown, since only "groups" that appear in the time frame are in the results.
 
+## Elastic Query Functions
+
+Elasitc replaces the deprecated logstash (ls) functions. It only works with Elastic v2+. It introduces two new types to allow for greater flexibility in querying. The ESIndexer type generates indexes to query (based on the date range). There are now different functions to generate indexers for people with different configurations. The ESQuery type is generates elastic queries so you can filter your results. By making these new types, new Indexers and Elastic queries can be added over time.
```

Comment:
so now we can also use these ES functions for any ES querying, without being tied to particular logstash schema? can this be clarified? are there any known limitations that we may want to document?